K2Bridge.Tests.End2End/KustoFunctionsTests.cs

```diff
@@ -43,6 +43,9 @@ public async Task FieldCaps_WithFunction_ReturnsResponseWithFunctions()
         {
             var fieldCaps = await K2Client().FieldCaps(functionFullName);
             var expected = JObject.Parse(@"{
+              ""indices"": [
+                ""fn_countries_and_airports""
```

Comment:
you can  use functionFullName
(which  is in the form of  db:fn_name, so maybe take the last part.

Reply:
👍🏻